lib/ansible/plugins/doc_fragments/oracle.py

```diff
@@ -0,0 +1,75 @@
+class ModuleDocFragment(object):
+    DOCUMENTATION = """
+    requirements:
+        - "python >= 2.7"
+        -  Python SDK for Oracle Cloud Infrastructure U(https://oracle-cloud-infrastructure-python-sdk.readthedocs.io)
+    notes:
+        - For OCI python sdk configuration, please refer to
+          U(https://oracle-cloud-infrastructure-python-sdk.readthedocs.io/en/latest/configuration.html)
+    options:
+        config_file_location:
+            description:
+                - Path to configuration file. If not set then the value of the OCI_CONFIG_FILE environment variable,
+                  if any, is used. Otherwise, defaults to ~/.oci/config.
+            required: false
+        config_profile_name:
+            description:
+                - The profile to load from the config file referenced by C(config_file_location). If not set, then the
+                  value of the OCI_CONFIG_PROFILE environment variable, if any, is used. Otherwise, defaults to the
+                  "DEFAULT" profile in C(config_file_location).
+            required: false
+            default: "DEFAULT"
```

Comment:
To improve the quality of the docs please add `type:` for all options in docs_fragments and the modules themselves.
```suggestion
            default: "DEFAULT"
            type: str
```

Reply:
added the type